backend/src/main/java/com/soen341/instagram/controller/RegistrationController.java

```diff
@@ -1,8 +1,42 @@
 package com.soen341.instagram.controller;
 
+import java.text.ParseException;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.soen341.instagram.dto.AccountDTO;
+import com.soen341.instagram.exception.account.BadRequestException;
+import com.soen341.instagram.model.Account;
+import com.soen341.instagram.service.impl.RegistrationService;
+
 @RestController
-public class RegistrationController {
+public class RegistrationController
+{
+
+	@Autowired
+	private ModelMapper modelMapper;
+
+	@Autowired
+	private RegistrationService registrationService;
 
+	@PostMapping(value = "Account/register")
```

Comment:
Maybe just `/register` ? If not, `Account` should probably be lowercase

Reply:
I'd keep `account` since it specify that this request will modify the ressource `account`
I'll put it lowercase.